docs/recipes/accounts.md

```diff
@@ -38,5 +38,75 @@ the amount of data you intend to store in the account.
   </CodeGroupItem>
 </CodeGroup>
 
+## Program Derived Address
+
+[Prgoram dervied address (PDA)][3] is like a normal address but
```

Comment:
This information is great. I am hoping to move all this kind of stuff into the bigger ingredient for PDAs, linking to the reference. Goot to keep for now

Reply:
Also a few nits on this:
Should be 
`Program derived address(PDA) is like a normal address with the following differences:`

Reply:
fixed at [here](https://github.com/solana-dev-adv/solana-cookbook/pull/37/commits/055c1d42948c4ad45c407d5986e70889926fc68e)